src/RcppExports.cpp

```diff
@@ -49,13 +49,13 @@ BEGIN_RCPP
 END_RCPP
 }
 // c_timestamp_encode
-RawVector c_timestamp_encode(double seconds, uint32_t nanoseconds);
+RawVector c_timestamp_encode(double seconds, u_int32_t nanoseconds);
```

Comment:
I would have thought `uint32_t`  was more common / standard?  I guess you like how `u_` sticks out?